### ISSUE TITLE
Assigning Locations to a company. 

### ISSUE THREAD

**Issue by victoriawhitmore** [grokability/snipe-it#7131]
**Server (please complete the following information):** Hosted
 - Snipe-IT Version - v4.7.4 - build 4107
 - OS: [e.g. Ubuntu, CentOS] - N/A
 - Web Server: [e.g. Apache, IIS] - N/A
 - PHP Version - N/A

**Is your feature request related to a problem? Please describe.**
I would like to be able to assign a location to a company so that when you use the multiple companies support to restrict users to only seeing assets in their company that this would also be the case for locations.  At the moment when I add locations for users to view they can also see the locations for all other companies rather than for only their company. 

**Describe the solution you'd like**
I would like to be able to assign a location to a company to make use of the multiple companies support feature to restrict users from seeing locations from other companies. 

Thank you for taking the time to hear my request. 

Kind Regards,

**Comment by victoriawhitmore** [comment]
Please keep this open as I would still like to see this added in future updates.

**Comment by victoriawhitmore** [comment]
Please keep this open as I would still like to see this added as a feature.

**Comment by victoriawhitmore** [comment]
Please keep this open as I would still like to see this added as a feature.

**Comment by victoriawhitmore** [comment]
Yes this is still relevant, it would be nice if we could assign locations to a company.

**Comment by victoriawhitmore** [comment]
yes still relevant

**Comment by victoriawhitmore** [comment]
still relevant

**Comment by victoriawhitmore** [comment]
Still Relevant

**Comment by piersandro** [comment]
Yes it is still rilevant. I would like to have the ability to assign locations to single companies.

**Comment by vcordes79** [comment]
Please merge PR #10705 
This missing functionality is the only thin stopping us from using Snipe IT

**Comment by snipe** [comment]
This was added a while back
